projects/storefrontlib/src/lib/cart/cart-details/container/cart-details.component.html

```diff
@@ -3,14 +3,11 @@
     <ng-container *ngIf="(entries$ | async) as entries">
       <div class="row">
         <div class="col-md-12 col-lg-9">
```

Comment:
I have a question about this. Technically one of the goals is to simplify the DOM to allow more configurability and extensibility. For example I noticed it's quite common to have the order-summary under the item list even on large screen.

The thing is that these bootstrap classes don't really allow much customization from the client's point of view. To make it more extendable/customizable we would have to make some changes to the html markup and create some custom style.

I know the bootstrap removal is not part of this refactoring but I just wanted to raise awareness about this and put it on the radar. What do you think?

Reply:
That's true regarding client's customizations. Off the top of my head: this will have to be approached either with cxOutlets for replacing with a very custom arrangement or with what Tobias is proposing as "agnostic" approach, to extend whatever is in place as ui framework.
For now, as long as we keep Bootstrap, there's no way around this, since it need a row div as parent for the columns, and then the columns need at least one value, but could have many, as to how they will distribute throughout media queries.

Reply:
Yes agreed that this is good. We will have more work about everything bootstrap soon so this is resolved.